### PR TITLE
Added Manifest Repository, mappers, type handler, service and the Model

### PR DESCRIPTION
**Pull request by paulinembabu**
- Added Manifest Model 
- Added Manifest Generated Mapper and Custom mapper
- Added Manifest Repository 
- Added Manifest TypeHandler 
- Added Manifest Service

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -3,3 +3,7 @@
 	url = https://github.com/OpenSRP/opensrp-server-configs
 	branch = master
 
+[submodule "opensrp-server-configs"]
+	path = opensrp-server-configs
+	url = https://github.com/OpenSRP/opensrp-server-configs
+	branch = master
```
Comment (githengi) [inline .gitmodules:9]: This is submodule is being duplicated

File: src/main/java/org/opensrp/repository/postgres/ManifestRepositoryImpl.java [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    }
+
+    @Nullable
+    @Override
+    public org.opensrp.domain.postgres.Manifest getManifest(String id) {
+        if (StringUtils.isBlank(id)) {
+            return null;
+        }
+        ManifestExample manifestExample = new ManifestExample();
+        List<org.opensrp.domain.postgres.Manifest> manifestList = manifestMapper.selectByExample(manifestExample);
```
Comment (githengi) [inline src/main/java/org/opensrp/repository/postgres/ManifestRepositoryImpl.java:121]: This is not filtering by Id params

File: src/main/java/org/opensrp/repository/postgres/PractitionerRepositoryImpl.java [hunk 3]
```diff
@@ -81,10 +81,6 @@ public void add(Practitioner practitioner) {
             return; // practitioner already added
         }
 
-        if (getPractitionerByUserId(practitioner.getUserId()) != null) {
-            return; // practitioner with this user id already added
-        }
-
```
Comment (githengi) [inline src/main/java/org/opensrp/repository/postgres/PractitionerRepositoryImpl.java:87]: This should be reverted

File: src/main/java/org/opensrp/repository/postgres/ManifestRepositoryImpl.java (hunk 2 above)
Comment (ekigamba) [reply src/main/java/org/opensrp/repository/postgres/ManifestRepositoryImpl.java:121]: Remove it since it's not being used

**Pull request merged**